lib/rucio/core/message.py

```diff
@@ -107,18 +106,25 @@ def retrieve_messages(bulk=1000, thread=None, total_threads=None, event_type=Non
         # Step 1:
         # MySQL does not support limits in nested queries, limit on the outer query instead.
         # This is not as performant, but the best we can get from MySQL.
+        # FIXME: SQLAlchemy generates wrong nowait MySQL8 statement for MySQL5
+        #        Remove once this is resolved in SQLAlchemy
```

Comment:
I thought this was solved by downgrading SQLAlchemy to 1.3.17?

Reply:
correct, will take this out.

Reply:
still in there? anything wrong?

Reply:
yeah, seems like this particular statement still requires this. not exactly sure why. the other NOWAIT errors all across the core are gone though.